lib/src/tasks/test/cli.dart

```diff
@@ -46,7 +46,11 @@ class TestCli extends TaskCli {
         abbr: 'p',
         allowMultiple: true,
         help:
-            'The platform(s) on which to run the tests.\n[vm (default), dartium, content-shell, chrome, phantomjs, firefox, safari]');
+            'The platform(s) on which to run the tests.\n[vm (default), dartium, content-shell, chrome, phantomjs, firefox, safari]')
+    ..addOption('test-name',
+        abbr: 'n',
```

Comment:
We should be consistent with the `test` package, and they name the option `name`:

```
-n, --name                     A substring of the name of the test to run.
                               Regular expression syntax is supported.
```